### PR TITLE
test: action not registered on frontend

### PR DESCRIPTION
**Pull request by carlossteinzup**
### Related Issues

https://github.com/ZupIT/beagle/issues/500
https://github.com/ZupIT/beagle/issues/1006

### Description and Example

Adds tests to unregistered actions.
When an action that is not registered is triggered the application should not crash.
At the moment it does nothing when an unregistered action is triggered.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/actions/ShowToast.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.automatedtests.actions
+
+import br.com.zup.beagle.annotation.RegisterAction
+import br.com.zup.beagle.annotation.RegisterWidget
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/actions/ShowToast.kt:20]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/3660e7d1961da702bd050edecd373a7d08526a86/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/actions/ShowToast.kt#L20"></span>The import ‘br.com.zup.beagle.annotation.RegisterWidget’ is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/actions/ShowToast.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import br.com.zup.beagle.widget.Widget
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/actions/ShowToast.kt:21]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/3660e7d1961da702bd050edecd373a7d08526a86/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/actions/ShowToast.kt#L21"></span>The import ‘br.com.zup.beagle.widget.Widget’ is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/ActionNotRegisteredScreenBuilder.kt [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.automatedtests.builders
+
+import br.com.zup.beagle.automatedtests.actions.ShowToast
+import br.com.zup.beagle.widget.action.Alert
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/ActionNotRegisteredScreenBuilder.kt:20]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/3660e7d1961da702bd050edecd373a7d08526a86/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/ActionNotRegisteredScreenBuilder.kt#L20"></span>The import ‘br.com.zup.beagle.widget.action.Alert’ is unused.</td>
    </tr>
  </tbody>
</table>

File: backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/ActionNotRegisteredScreenBuilder.kt [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import br.com.zup.beagle.widget.context.ContextData
```
Comment (botbeaglezup) [inline backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/ActionNotRegisteredScreenBuilder.kt:21]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/3660e7d1961da702bd050edecd373a7d08526a86/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/ActionNotRegisteredScreenBuilder.kt#L21"></span>The import ‘br.com.zup.beagle.widget.context.ContextData’ is unused.</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: iOS/Tests/AutomatedTestsUITests/Steps/ActionNotRegisteredSteps.swift [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        // MARK: - Then
+
+        Then(#"^nothing happens and the "([^\"]*)" title still appears on screen$"#) { args, _ -> Void in
+            let text = args![0]
+            XCTAssertTrue(self.application.staticTexts[text].exists)
+        }
+    }
+}
+
+fileprivate struct Constants {
```
Comment (botbeaglezup) [inline iOS/Tests/AutomatedTestsUITests/Steps/ActionNotRegisteredSteps.swift:55]: <table data-meta="generated_by_ios">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/aa1014888b0e9d3ba23adce597fa05958b4744bf/iOS/Tests/AutomatedTestsUITests/Steps/ActionNotRegisteredSteps.swift#L55"></span>Prefer <code>private</code> over <code>fileprivate</code> declarations.<br />
<code>private_over_fileprivate</code> <code>ActionNotRegisteredSteps.swift:55</code></td>
    </tr>
  </tbody>
</table>

**Review by viniciusguardieirozup — DISMISSED** [review]

**Review by FredericoNoce — DISMISSED** [review]

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/df5f17a610124cd6ef2dc4e3a16afedd1ebe862a/backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/controllers/GeneralController.kt#L48">backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/controllers/GeneralController.kt#L48</a> - Class ‘GeneralController’ with ‘13’ functions detected. Defined threshold inside classes is set to ‘13’</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_backend">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by vitorbassozup — APPROVED** [review]

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by uziasferreirazup — APPROVED** [review]

**Review by viniciusguardieirozup — APPROVED** [review]

**Review by joaoabreuzup — APPROVED** [review]

**Pull request merged**